leaf-ui/js/object/evoObjects.js

```diff
@@ -377,20 +386,22 @@ class EVO {
                 curr.attributes.type !== 'basic.Resource') {
                 continue;
             }
-            intention = model.getIntentionByID(curr.attributes.nodeID);
-            initSatVal = intention.getInitialSatValue();
-            if(!analysisResult.isPathSim){   
-                if (initSatVal === '(no value)') {
-                    curr.attr('.satvalue/text', '');
-                    curr.attr({text: {fill: 'black',stroke:'none','font-weight' : 'normal'}});
-
+            intention = elements[i].get('intention');
+            initSatVal = intention.getUserEvaluationBBM(0).get('assignedEvidencePair'); 
+            if(analysisResult !== null){
+                if(!analysisResult.get('isPathSim')){   
+                    if (initSatVal === '(no value)') {
+                        curr.attr('.satvalue/text', '');
+                        curr.attr({text: {fill: 'black',stroke:'none','font-weight' : 'normal'}});
+    
+                    }else{
```

Comment:
Delete extra line above here and add a space between `else` and `}{`

Same for line 401